configs/microblaze-zephyr-eabi.config

```diff
@@ -0,0 +1,63 @@
+CT_CONFIG_VERSION="3"
```

Comment:
This file must be named `microblaze-zephyr-elf.config`, not `microblaze-zephyr-eabi.config`.

Also, since Microblaze toolchain defaults to big endian and you want little endian, the target name should be `microblazeel-zephyr-elf`, and therefore the config file name should be `microblazeel-zephyr-elf.config`

Reply:
> since Microblaze toolchain defaults to big endian and you want little endian

Yes, had a lot of trouble with that during linking stages. Thank you! Making the changes.